ambry-store/src/main/java/com/github/ambry/store/BlobStoreStats.java

```diff
@@ -160,6 +164,12 @@ static StatsSnapshot convertStoreUsageToContainerStatsSnapshot(Map<Short, Map<Sh
       queueProcessor = new QueueProcessor();
       shortLiveTaskScheduler.scheduleAtFixedRate(queueProcessor, 0, queueProcessingPeriodInMs, TimeUnit.MILLISECONDS);
     }
+    if (enableAsyncGetValidSize) {
+      ScheduledExecutorService scheduler = Executors.newSingleThreadScheduledExecutor();
+      TimeRange timeRange = new TimeRange(System.currentTimeMillis(), getBucketSpanTimeInMs());
```

Comment:
Should this time range be a constant or should it be refreshed to the current time whenever ValidDataSizeCollector.run is called?

Reply:
Thanks for catching that, fixed it.